### PR TITLE
Use delete objects in lakectl rm recursive

### LINKED ISSUE #6914: `lakectl fs rm --recursive` - improve efficiency
When executing `lakectl fs rm --recursive`, the objects are deleted one by one using the `deleteObject` API endpoint instead of grouping the objects to a single call using the `deleteObjects` API call.

### PR DESCRIPTION
**Pull request by lynnro314**
Closes #6914

### CODE REVIEW

File: cmd/lakectl/cmd/fs_rm.go [hunk 1]
```diff
@@ -93,12 +88,33 @@ var fsRmCmd = &cobra.Command{
 	},
 }
 
-func deleteObjectWorker(ctx context.Context, client apigen.ClientWithResponsesInterface, paths <-chan *uri.URI, errors chan<- error, wg *sync.WaitGroup) {
+func deleteObjectWorker(ctx context.Context, client apigen.ClientWithResponsesInterface, repository, branch string, paths <-chan string, errors chan<- error, wg *sync.WaitGroup) {
 	defer wg.Done()
-	for pathURI := range paths {
-		err := deleteObject(ctx, client, pathURI)
+	objs := make([]string, 0, 1000)
+	fmt.Println("deleteObjectWorker")
```
Comment (nopcoder) [inline cmd/lakectl/cmd/fs_rm.go:94]: debug print

File: cmd/lakectl/cmd/fs_rm.go [hunk 2]
```diff
@@ -93,12 +88,33 @@ var fsRmCmd = &cobra.Command{
...
+	for objPath := range paths {
+		objs = append(objs, objPath)
+		if len(objs) >= 1000 {
```
Comment (nopcoder) [inline cmd/lakectl/cmd/fs_rm.go:97]: delete chunk size as const or flag with default

File: cmd/lakectl/cmd/fs_rm.go [hunk 3]
```diff
@@ -93,12 +88,33 @@ var fsRmCmd = &cobra.Command{
...
+			err = RetrieveError(resp, err)
+			if err != nil {
+				rmErr := fmt.Errorf("rm objects - %w", err)
+				errors <- rmErr
+			}
+			clear(objs)
+		}
+	}
+	if len(objs) > 0 {
+		// err := deleteObject(ctx, client, pathURI)
```
Comment (nopcoder) [inline cmd/lakectl/cmd/fs_rm.go:111]: remove comment

**Review by johnnyaug — APPROVED** [review]
Nice!

**Comment by arielshaqed** [comment]
1. This is deserving of "include-changelog", it improves performance!  I went ahead and changed that label.
1. We may want to test `lakectl fs rm` in `lakectl_test.go`.

**Pull request merged**